src/connectors/panoptic/panoptic.controllers.ts

```diff
@@ -103,47 +146,54 @@ export async function estimateGas(
 export async function calculateDelta(
   panopticish: Panoptic,
   req: CalculateDeltaRequest
-): Promise<any> {
+): Promise<CalculateDeltaResponse> {
   const result = await panopticish.calculateDelta(req.PRICE, req.RANGE, req.STRIKE);
-  return result;
+  return {
+    delta: result
+  };
 }
 
 export async function calculateGamma(
   panopticish: Panoptic,
   req: CalculateGammaRequest
-): Promise<any> {
+): Promise<CalculateGammaResponse> {
   const result = await panopticish.calculateGamma(req.PRICE, req.RANGE, req.STRIKE);
-  return result;
+  return {
+    gamma: result
+  };
 }
 
-
 // Subgraph interactions
 
 export async function queryOpenPositions(
   ethereumish: Ethereumish,
   panopticish: Panoptic,
   req: QueryOpenPositionsRequest
-): Promise<any> {
+): Promise<QueryOpenPositionsResponse> {
   const { wallet } = await txWriteData(ethereumish, req.address);
   const result = await panopticish.queryOpenPositions(wallet);
-  return result;
+  return {
+    queryResponse: result
```

Comment:
For any specialized subgraph query, I thought it would be good to hang on to the full query response, then unpack the values of interest into their own variables. In this case, yes, we can definitely unpack the positions part and give it a variable. I added an output named "positions". Though, here, I think that is just the same as the full query response. If this seems too redundant we can play with it more.  c074db31846151c662e5c51b1e4bbc0f61fa855a

Reply:
I changed this to nicely unpack the full/open/closed position lists: e7a19c0406f1a8c15ef8e70522e946029750691f